api/config/v1alpha1/envoyproxy_types.go

```diff
@@ -86,6 +86,13 @@ type KubernetesResourceProvider struct {
 	//
 	// +optional
 	EnvoyDeployment *KubernetesDeploymentSpec `json:"envoyDeployment,omitempty"`
+
+	// EnvoyService defines the desired state of the Envoy service resource.
+	// If unspecified, default settings for the manged Envoy service resource
+	// are applied.
```

Comment:
should mention what's the default value.

Reply:
its `nil`

Reply:
added a follow up commit to address this, ptal